pallets/payment/src/lib.rs

```diff
@@ -56,6 +56,9 @@ pub mod pallet {
 		/// Buffer period - number of blocks to wait before user can claim canceled payment
 		#[pallet::constant]
 		type CancelBufferBlockLength: Get<Self::BlockNumber>;
+		/// Minimum amount to be transferred in a payment
+		#[pallet::constant]
+		type MinPaymentAmount: Get<BalanceOf<Self>>;
```

Comment:
Don't think it's needed if we use [`MultiCurrency::minimum_balance`](https://github.com/open-web3-stack/open-runtime-module-library/blob/master/traits/src/currency.rs#L42) to determine if the transaction should go through

Reply:
That makes sense, but the downside is that the pallet becomes less generic/configurable. For a use case where the chain requires ExistentialDeposit=0, they will have to fork the pallet. Also I feel its better for visibility to explicitly set the minimum value ie. when the pallet is imported this value is forced to be set and not buried inside as implementation detail. wdyt?

Reply:
But then you will need something more like [this](https://github.com/open-web3-stack/open-runtime-module-library/blob/master/tokens/src/lib.rs#L199) where the balance can be determined by token. That said I don't think it's going to be less generic since the minimum balance comes from the already generic `MultiCurrency` trait that the user controls, e.g. we have the `ExistentialDeposits` from the tokens pallet configured in our runtime and if we wanted to we can wrap the tokens pallet in some adapter that overrides the minimum balance requirement.
As I commented in the review, I don't know how the minimum payment helps the spam prevention more than the intrinsic fee of the transaction but for sure we should make sure that a payment made to a new account has the minimum necessary to not be reaped